client/js/events/listings/index.js

```diff
@@ -24,16 +24,16 @@ module.exports = pathPrefix => {
 
   const listings = (key, state) => {
     const resp$ = state[key]
-    const mixin = content =>
+    const sectionForEventType = content =>
```

Comment:
`mixin` isn't a helpful name. I don't know that `sectionForEventType` is a fantastic name but it's better than `mixin`